dada2_chimera_taxa.R

```diff
@@ -0,0 +1,264 @@
+#!/usr/bin/env Rscript
+
+# Read in package to read in command-line options.
+library("optparse")
+
+version <- "1.0"
+
+option_list <- list(
+
+  make_option(c("-i", "--input"), type="character", default=NULL,
+              help="Location of input .RDS file (required)." , metavar="path"),
+  
+  make_option(c("-r", "--refFasta"), type="character", default=NULL,
+              help="Location of reference fasta to use for taxa assignment (required).", 
+              metavar="path"),
+  
+  make_option(c("-s", "--ref_species"), type="character", default=NULL,
+              help="Location of reference fasta to use for species assignment (required).", 
```

Comment:
If user sets --skip_species to TRUE, is --ref_species still required?

Reply:
No it isn't, but I'll make that clearer.